end_to_end/4-deploy-run-inference-e2e.ipynb

```diff
@@ -0,0 +1,879 @@
+{
```

Comment:
<p>Got an error (on SM instance)</p><pre class="ql-syntax" spellcheck="false">IndexError                                Traceback (most recent call last)
&lt;ipython-input-18-47042b69d948&gt; in &lt;module&gt;
      1 primary_container = {'ModelPackageName': second_model_package['ModelPackageArn']}
      2 endpoint_config_name=f'{model_2_name}-endpoint-config'
----&gt; 3 existing_configs = sagemaker_boto_client.list_endpoint_configs(NameContains=endpoint_config_name, MaxResults = 30)['EndpointConfigs'][0]['EndpointConfigName']
      4 
      5 if not existing_configs:

IndexError: list index out of range
</pre><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>apologies, I do not receive this error.</p>

Reply:
<p>PLs make sure the endpoint config name exists</p>

Reply:
fixed